templates/alertmanagerconfig.go

```diff
@@ -115,6 +115,45 @@ var AlertmanagerConfigTemplate = promv1a1.AlertmanagerConfig{
 				To:           "",
 				AuthUsername: "",
 				AuthPassword: &corev1.SecretKeySelector{Key: "", LocalObjectReference: corev1.LocalObjectReference{Name: ""}},
+				Headers: []promv1a1.KeyValue{{
+					Key:   "subject",
+					Value: `{{ range .Alerts.Firing }}{{ range .Labels.SortedPairs }}{{ if eq .Name "severity" }}{{ .Value }}{{ end }}{{ end }}{{ end }} alert! Action Required on your managed OpenShift cluster`,
+				}},
+				HTML: `<!DOCTYPE html>
+				<html>
+					<head>
+						<title>Hello</title>
+					</heah>
+					<body itemscope itemtype="http://schema.org/EmailMessage">
+						<p>
+							Hello!
+							<br><br>
+							This notification is for your OpenShift managed cluster running OpenShift Data Foundation.
+							<br><br>
+							{{ range .Alerts.Firing }}
+								{{ range .Labels.SortedPairs }}
+									{{ if eq .Name "alertname" }}
+											Alert: <strong>{{ .Value }}</strong>
+									{{ end }}
+								{{ end }}
+								<br>
+								{{ range .Annotations.SortedPairs }}
+									{{ if eq .Name "description" }} 
+										<strong>{{ .Value }}</strong>
+									{{ end }}
+								{{ end }}
+							<br><br>
+							{{ end }}
+							Please log in to your cluster to take appropriate action
+							<br>
+							If you have any questions, please <a href="https://access.redhat.com/support/contact/technicalSupport/">contact us</a>. Review the <a href="https://access.redhat.com/support/policy/support_process">support process</a> for guidance on working with Red Hat support.
+							<br><br>
+							Thank you for choosing Red Hat OpenShift Data Foundation,
+							<br>
+							ODF SRE
+						</p>
+					</body>
+				</html>`,
```

Comment:
Can we move the HTML into its own `.html` file and read it into the template? something like `customer-notification.html` (can stay in the templated folder)

It will give us the following benefits:
1. A more readable and clean template
2. HTML editing support from our editors
3. Will open the door to dynamically having different templates for each alert in the future (by introducing selection code to the HTML file itself)

Reply:
Done